Tools/machines/lumi-csc/lumi_warpx.profile.example

```diff
@@ -0,0 +1,51 @@
+# please set your project account
+#export proj=<yourProject>
+
+# optional: just an additional text editor
+module load nano
+
+# required dependencies
```

Comment:
We could use their spack instance for all our deps :)

https://docs.lumi-supercomputer.eu/software/installing/spack/